content/en/docs/tasks/debug-application-cluster/debug-pod-replication-controller.md

```diff
@@ -123,6 +123,8 @@ As an example, to look at the logs from a running Cassandra pod, you might run:
 kubectl exec cassandra -- cat /var/log/cassandra/system.log
 ```
 
+Besides, you can also try instantiating an [ephemeral container](/docs/concepts/workloads/pods/ephemeral-containers/), which is an alpha feature introduced in version 1.16, in the exist pod to inspect its state and run arbitrary commands.
```

Comment:
Hi @Colstuwjx 
The language here isn't quite idiomatic English.
How about:
```markdown
If your cluster supports it, ou can also try adding an
[ephemeral container](/docs/concepts/workloads/pods/ephemeral-containers/) into an
existing Pod. You can the new temporary container to run arbitrary commands; for example,
to diagnose problems inside the Pod. See the page about ephemeral containers for more
details, including feature availability.
```

?